components/template-mgt/org.wso2.carbon.identity.template.mgt/src/main/java/org/wso2/carbon/identity/template/mgt/TemplateMgtConstants.java

```diff
@@ -43,6 +43,9 @@
     public static final String HEADER_CONTENT_TYPE = "Content-Type";
     public static final String DEFAULT_RESPONSE_CONTENT_TYPE = APPLICATION_JSON;
     public static final String STATUS_BAD_REQUEST_MESSAGE_DEFAULT = "Bad Request";
+    public static final String FORBIDDEN = "Forbidden";
+    public static final String FORBIDDEN_ERROR_CODE = "403";
+    public static final String FORBIDDEN_ERROR_MESSAGE= "Template-mgt API is not supported";
```

Comment:
shall we say Template-mgt API is restricted or something similar

Reply:
Fixed https://github.com/wso2/carbon-identity-framework/pull/2996/commits/59a224166a3a049623549bb53ac550322a2eaee0